packages/plugins/rubygem-foreman_snapshot_management/rubygem-foreman_snapshot_management.spec

```diff
@@ -31,6 +27,7 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 BuildRequires: npm(@babel/core) >= 7.7.0
 BuildRequires: npm(@babel/core) < 8.0.0
 BuildRequires: npm(@theforeman/builder) >= 4.0.0
+BuildRequires: npm(@theforeman/builder) < 5.0.0
```

Comment:
This needs to be removed (preferably also in your `package.json`, so it won't come back)

Reply:
We have an open PR for this. We should merge it, eventually.